staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/nonblockinggrpcserver.go

```diff
@@ -78,7 +78,10 @@ func startGRPCServer(logger klog.Logger, grpcVerbosity int, unaryInterceptors []
 		defer s.wg.Done()
 		err := s.server.Serve(listener)
 		if err != nil {
-			logger.Error(err, "GRPC server failed")
+			logger.Error(err, "gRPC server failed to serve")
```

Comment:
Let's log here only if we don't pass on the error. Otherwise it might get logged twice.